tests/upgrades_tests.cairo

```diff
@@ -0,0 +1,114 @@
+use traits::Into;
+use traits::TryInto;
+
+use starknet::{ContractAddress, storage_access::storage_address_from_base};
+
+use konoha::contract::IGovernanceDispatcher;
+use konoha::contract::IGovernanceDispatcherTrait;
+use konoha::proposals::IProposalsDispatcher;
+use konoha::proposals::IProposalsDispatcherTrait;
+use konoha::upgrades::IUpgradesDispatcher;
+use konoha::upgrades::IUpgradesDispatcherTrait;
+use konoha::constants;
+
+use snforge_std as snf;
```

Comment:
we import prank from snforge_std everywhere else, so it would be nice to be consistent and use `snforge_std::{CheatTarget,ContractClassTrait,prank, get_class_hash}` etc

Reply:
Ok I have updated the source code